src/graphql/mirror.js

```diff
@@ -35,6 +36,39 @@ export class Mirror {
     this._initialize();
   }
 
+  /**
+   * Embed the GraphQL schema into the database, initializing it for use
+   * as a mirror.
+   *
+   * We store the data for an object across three tables, depending on
```

Comment:
This blcok is confusing. Are there three tables for each kind of object? or are there three tables overall? please be more specific (an example would be nice)

Reply:
It would be nice if we start at a high level: the database has one connections table for all objects' connections, one links table, and then, for each object type, one type-specific table that has columns for every primitive field. (assuming this is true). getting in to the columnar structure for the connections and links table would be great. also, how does the meta table fit into this precisely? does every object type have its own meta entry?